PlayCover/View/AppsView.swift

```diff
@@ -88,7 +79,7 @@ struct AppsView: View {
 						}
 					}
 					.padding(.top, 16).padding(.bottom, bottomPadding + 16)
-					.animation(.spring())
+                    .animation(.spring())
```

Comment:
Why was the indentation changed here?

Reply:
Indentendation wasn't intentionally changed. In Xcode it doesn't show that anything changed.

Reply:
weird

Reply:
There were no lint violations, so it should be alright.